source/_components/binary_sensor.wirelesstag.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "WirelessTag Binary Sensor"
+description: "Instructions on how to integrate Wireless Tags sensors within Home Assistant."
+date: 2018-03-26 21:49
+comments: false
+sidebar: true
+sharing: true
+footer: true
+logo: wirelesstag.png
+ha_category: Binary Sensor
+ha_iot_class: "Local Push"
+---
+
+To get your [wirelesstag.net](http://wirelesstag.net) binary sensors working within Home Assistant, please follow the instructions for the general [WirelessTag component](/components/wirelesstag).
+
+To enable tags set up with your [wirelesstag.net](http://wirelesstag.net) account, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+binary_sensor:
+  - platform: wirelesstag
+    monitored_conditions:
+      - presence
+      - door
+      - low_battery
+```
+
+{% configuration %}
+  monitored_conditions:
+    description: The conditions types to monitor; valid values are specified below
+    required: true
+    type: list
+{% endconfiguration %}
+
+The following conditions can be monitored:
+
+* (`presence`): On means in range, Off means out of range.
+* (`motion`): On when when movement was detected, Off when clear.
+* (`door`): On when when door is open, Off when door is closed.
```

Comment:
`On when when door is open, Off when door is closed.` -> `On when a door is open, Off when the door is closed.`